src/hooks/useRouting.ts

```diff
@@ -1,15 +1,8 @@
-import { useRouteMatch } from 'react-router'
-
-type ParamsType = {
-  params: {
-    gardenAddress: string
-    networkType: string
-  }
-} | null
+import { useRouter } from 'next/router'
 
 export function useGardenRoute() {
-  const match: ParamsType = useRouteMatch('/:networkType/garden/:gardenAddress')
```

Comment:
Can we double check we are still handling the routes correctly?

Reply:
Maybe create some jest file test?

Reply:
Yes, `networkType` and `gardenAddress` params are available for us from the `useRouter` hook from `next/router`.

This will remove the dependency of performing a route match like what we are doing.